src/lib/utils/thorchain/memo/addFinalAssetContractToMemo.test.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, expect, it } from 'vitest'
+
+import { addFinalAssetContractToMemo } from './addFinalAssetContractToMemo'
+
+describe('addFinalAssetContractToMemo', () => {
```

Comment:
nitpick: `final asset address` terminology over `final asset contract` across this PR

Reply:
in [2a6fc6a](https://github.com/shapeshift/web/pull/7146/commits/2a6fc6a57521f17fb2fa9ee67d3d46f222c6e53c)